README.md

```diff
@@ -1,2 +1,36 @@
 # TDD Exercise
 
+In this exercise you will write a method and set of tests in TDD fashion which calculates a hand's BlackJack score.
+
+## About BlackJack
+
+In the card name BlackJack each card has a value.
+-  Number cards (2-10) carry the card's numeric value.
+-  Face cards on the other hand ("Jack", "Queen", "King") have a value of 10.
+-  Aces (1) can have a value of either 1 or 11, whichever will get the hand closest to 21 without going over.
+
```

Comment:
In the above paragraph, it talks about the game Blackjack in the real-world scenario and its rules.

In the below paragraph, it switches to talking about it as a program (it's described that the hand is the array `[3, "King", 1]`. Since this section is about Blackjack's rule in the real-life scenario/business scenario, and not about implementation, it may help to just say "For example, if my hand has three cards, a 3, a King, and a 1, my Blackjack score is 14 (3 + 10 + 1). If I have a two-card hand of a 1 and a Jack, then my score is 21 (11 +10)."